docs/troubleshooting.rst

```diff
@@ -28,3 +28,13 @@ The following is a list of commonly reported issues in Open Federated Learning (
 
    Yes! You can turn on verbose logging with :code:`fx -l DEBUG collaborator start` or :code:`fx -l DEBUG aggregator start`. This will give verbose information related to gRPC, bidirectional tensor transfer, and compression related information.  
 
+5. **Silent failures resulting from Out of Memory errors**
+
+   Observations:
+      * :code:`fx envoy` command terminates abruptly during the execution of training or validation loop due to the SIGKILL command issued by the kernel. 
+      * OOM error is captured in the kernel trace but not at the user program level.
+      * The failure is likely due to non-optimal memory resource utilization in the prior PyTorch version 1.3.1 & 1.4.0.
+      * Interactive API example fails with the PyTorch v1.3.1/v1.4.0 and CG memory restricted to 4GB but succeeds with PyTorch v1.11.0 and CG memory restricted to 768MB. 
```

Comment:
This line can be removed

Reply:
Shall I remove the entire observations section or just the line 37?

Reply:
Just line 37

Reply:
Got it. 
I too felt its too specific.